ci/bundle_custom_data.py

```diff
@@ -40,4 +40,5 @@
     "brats_mri_segmentation",
     "endoscopic_inbody_classification",
     "spleen_deepedit_annotation",
+    "spleen_ct_segmentation",
```

Comment:
should this list be updated?

Reply:
https://github.com/Project-MONAI/model-zoo/pull/394/files#diff-74b06043d5901ecb806cd7a85f3bd03a89b63c0e85ec3ca125c39ae4aada319fL35

Reply:
Hi @yiheng-wang-nv ,
     Actually the spleen_ct_segmentation bundle supports both ONNX-TensorRT and Torch-TensorRT ways of conversion. However the Torch-TensorRT way cannot achieve a decent speedup ratio. Therefore, for the short term usage, the ONNX-TensorRT way is chosen to export the model. But from the long term point of view, it will back to the Torch-TensorRT way finally (as long as it achieves a normal speedup ratio). So I think we'd better keep both TensorRT checks, in case there are some regression issues of Torch-TensorRT. What do you think about this.

Thanks,
Bin